Runtime/Model/Session/SessionEvent.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Backtrace.Unity.Json;
+using System.Collections.Generic;
+
+namespace Backtrace.Unity.Model.Session
+{
+    internal sealed class SessionEvent : EventAggregationBase
+    {
+        internal const string MetircsGroupName = "metric_group";

```

Comment:
Should probably be `MetricGroupName` (backend is singular, and this is also misspelled).

Reply:
solved, thanks!